docs/_static/llama-stack-spec.yaml

```diff
@@ -19,6 +19,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PaginatedRowsResult'
+        '400':
+          $ref: '#/components/responses/BadRequest400'
+        '429':
+          $ref: >-
+            #/components/responses/TooManyRequests429
```

Comment:
@ashwinb this one is called RateLimitError by Stainless. I guess it's just label but the SDK user will be using exception names defined here https://fburl.com/f2mpnlr6